src/EditorFeatures/VisualBasic/ImplementAbstractClass/ImplementAbstractClassCommandHandler.vb

```diff
@@ -41,8 +41,7 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.ImplementAbstractClass
                 Return Nothing
             End If
 
-            Dim service = document.GetLanguageService(Of IImplementAbstractClassService)()
```

Comment:
no need for language services.  nothing about this feature has any sort of VB or C# specific behavior...